projects/ion/src/lib/step/step.component.html

```diff
@@ -0,0 +1,57 @@
+<section class="steps-container" data-testid="ion-steps">
+  <div
+    [class]="'step ' + step.status"
+    [class.clickable]="
+      clickable &&
+      !disabled &&
+      step.status !== 'selected' &&
+      step.status !== 'error'
+    "
+    [class.disabled]="disabled"
+    [attr.data-testid]="'step-' + step.index + '-' + step.status"
+    *ngFor="let step of steps; let index = index"
+    (click)="goesTo(step.index)"
+  >
+    <div class="step-draw">
+      <div
+        class="line"
+        [class.bolded]="beforeStepIsChecked(index)"
+        *ngIf="lineVisibility('initial', step)"
+      ></div>
+
+      <div class="circle">
+        <span *ngIf="step.status !== 'checked'; else checkedStatus">
+          {{ step.index }}
+        </span>
+        <ng-template #checkedStatus>
+          <span>
+            <ion-icon type="check" size="24"></ion-icon>
+          </span>
+        </ng-template>
+      </div>
+
+      <div
+        class="line"
+        [class.bolded]="afterStepIsChecked(index)"
+        *ngIf="lineVisibility('final', step)"
```

Comment:
LineVisibility comment add-on

```suggestion
       *ngIf="step.index !== steps.length"
```